src/components/tooltip/TooltipDirective.js

```diff
@@ -462,10 +527,7 @@ goog.require('ga_topic_service');
                     $http.get(htmlUrl, {
                       timeout: canceler.promise,
                       params: {
-                        lang: $translate.use(),
-                        mapExtent: mapExtent.join(','),
-                        coord: (coordinate) ? coordinate.join(',') : undefined,
-                        imageDisplay: size[0] + ',' + size[1] + ',96'
+                        lang: $translate.use()
```

Comment:
It's the doc there  is only lang https://api3.geo.admin.ch/services/sdiservices.html#htmlpopup-resource .

are they hidden parameter?